lib/h5p.js

```diff
@@ -804,11 +804,11 @@ function removeUntranslatables(field, name) {
       }
     }
 
-    if (field.length === 0) {
+    if (field !== null && Object.keys(field).length === 0) {
```

Comment:
Object.keys(field).length === 0 check fails because it deletes non-translatable JSON objects in semantics, instead of replacing them with an empty JSON object, please change this before I can merge it in :)